src/js/view/components/nextuptooltip.js

```diff
@@ -114,14 +109,19 @@ define([
                 .on('click tap', this.playNext, this);
             // setup thumbnail
             this.img = element.getElementsByClassName('jw-nextup-thumbnail')[0];
-            this.image(this.loadThumbnail(nextUpItem.image));
+            if (nextUpItem.image) {
+              this.image(this.loadThumbnail(nextUpItem.image));
+              dom.addClass(this.img, 'jw-nextup-thumbnail-visible');
+            } else {
+              dom.removeClass(this.img, 'jw-nextup-thumbnail-visible');
+            }
 
             // set header
             this.header = element.getElementsByClassName('jw-nextup-header')[0];
             this.header.innerText = this._nextUpText;
             // set title
             this.title = element.getElementsByClassName('jw-nextup-title')[0];
-            this.title.innerText = nextUpItem.title || 'title';
+            this.title.innerText = nextUpItem.title || 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
```

Comment:
@johnBartos @madikarizma It's ok to see the behavior for now as this is a PR to the feature branch, not against feeds/master. I'll remove it before creating a PR for the ticket.